test/Makefile

```diff
@@ -15,11 +18,17 @@ build:
 
 # Build the project with coverage instrumentation
 build-coverage:
-	CARGO_INCREMENTAL=0 RUSTFLAGS="$(RUSTFLAGS_COVERAGE)" cargo build $(FEATURES)
+	CARGO_INCREMENTAL=0 RUSTFLAGS="$(RUSTFLAGS_COVERAGE)" cargo +nightly build $(FEATURES)
+
+build-image:
+	docker build -t libdroptestimage .
 
 # Test the project
-test: build
-	LIB_PATH=$(LIB_DOCKER) ./runner.py
+test: build build-image
+	docker kill $(docker ps -q) || true
+	docker network prune -f
+	docker container prune -f
```

Comment:
Can you include this one in the coverage as well? Possibly extracting it into a separate rule

Reply:
sure!

Reply:
added